assets/css/portfolio.css

```diff
@@ -278,6 +279,94 @@ section.about .img-wrap img {
 .about-content a.btn i {
   margin-right: 10px;
 }
+
+/*contact section*/
+
+.contact-me{
+  background: #000000;
+  color: #fff;
```

Comment:
wrong color scheme, please look at the figma file, it has the exact css you need